internal/server/data/schema.sql

```diff
@@ -215,65 +141,27 @@ CREATE TABLE providers (
     kind text,
     auth_url text,
     scopes text,
+    organization_id bigint,
     private_key text,
     client_email text,
     domain_admin_email text
 );
 
-CREATE SEQUENCE providers_id_seq
-    START WITH 1
-    INCREMENT BY 1
-    NO MINVALUE
-    NO MAXVALUE
-    CACHE 1;
-
-ALTER SEQUENCE providers_id_seq OWNED BY providers.id;
-
 CREATE TABLE settings (
     id bigint NOT NULL,
     created_at timestamp with time zone,
     updated_at timestamp with time zone,
     deleted_at timestamp with time zone,
     private_jwk bytea,
     public_jwk bytea,
+    organization_id bigint,
     lowercase_min bigint DEFAULT 0,
     uppercase_min bigint DEFAULT 0,
     number_min bigint DEFAULT 0,
     symbol_min bigint DEFAULT 0,
     length_min bigint DEFAULT 8
 );
 
-CREATE SEQUENCE settings_id_seq
-    START WITH 1
-    INCREMENT BY 1
-    NO MINVALUE
-    NO MAXVALUE
-    CACHE 1;
-
-ALTER SEQUENCE settings_id_seq OWNED BY settings.id;
-
-ALTER TABLE ONLY access_keys ALTER COLUMN id SET DEFAULT nextval('access_keys_id_seq'::regclass);
```

Comment:
Should we still have these alter table statements? We manually set the ID by they seem ok to have.

Reply:
I'll let @ssoroka comment as well, because I think he removed them originally.

I like removing them because we don't use them, and I think having these sequences could hide bugs from us in testing. I think it's better off if the column is set to `NOT NULL` without a sequence to force us to set values.